js/zb.js

```diff
@@ -686,11 +686,18 @@ module.exports = class zb extends Exchange {
         const market = this.market (symbol);
         const request = {
             'market': market['id'],
+            'symbol': market['id'],
```

Comment:
Do we have to send both the `market` and the `symbol` here ? I think it's one for spot, the other for swap, but not both.

Reply:
I made a new commit splitting these requests based on market type, let me know if I should use a different format for handling this